torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -7292,6 +7296,56 @@ def get_loss(model_output):
                     ):
                         self.assertEqual(p, p_static)
 
+        @skip_if_lt_x_gpu(2)
+        @unittest.skipIf(
+            BACKEND != "nccl" and BACKEND != "gloo",
+            "Only Nccl & Gloo backend support DistributedDataParallel",
+        )
+        def test_ddp_get_bucket_sizes(self):
+            torch.cuda.set_device(self.rank)
+            default_bucket_cap_mb = 25 * (1024 ** 2)
+            first_bucket_bytes_mb = dist._DEFAULT_FIRST_BUCKET_BYTES
+
+            class MyModel(nn.Module):
+                def __init__(self):
+                    super().__init__()
+                    self.model = nn.Sequential(nn.Linear(2, 4000, bias=False),
+                        *[
```

Comment:
Shall we fix lint errors before landing?

Reply:
yep, looks like there's a test failure as well